apps/src/dance/ai/utils/generateBlocks.ts

```diff
@@ -0,0 +1,8 @@
+import {BlockSvg, Workspace} from 'blockly';
+
+export const generateBlocks = (workspace: Workspace): [BlockSvg, BlockSvg] => {
```

Comment:
I think we should ensure that each new file has at least a small comment explaining what it's for.